includes/Modules/AdSense.php

```diff
@@ -403,7 +403,8 @@ function( Google_Service_Adsense_Alert $alert ) {
 	protected function parse_data_response( Data_Request $data, $response ) {
 		switch ( "{$data->method}:{$data->datapoint}" ) {
 			case 'GET:accounts':
-				return array_map( array( self::class, 'filter_account_with_ids' ), $response->getAccounts() );
+				$accounts = $response->getAccounts();
+				return array_filter($accounts,'filter_account_with_state_closed');
```

Comment:
You're removing the `array_map` call here, but we still need to keep that in here. We need both your new `array_filter` call _and_ the existing `array_map` call.

Also, please follow the WordPress coding standards, there need to be spaces in the appropriate locations.